.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,89 @@
+name: 🐞 Bug report
+description: Create a report about something that is not working
+labels: ["untriaged"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting **non-security** bugs and feature requests.
+        For general questions, consider posting to [Discussions](https://github.com/microsoft/dotnet-framework-docker/discussions) or [Stack Overflow](https://stackoverflow.com).
+
+        If you're reporting the presence of a disclosed security vulnerability, such as a CVE reported in one of our container images, please follow our [vulnerability reporting guidelines](https://github.com/dotnet/dotnet-docker/blob/main/documentation/vulnerability-reporting.md).
```

Comment:
That doc is very much specific to the images at dotnet-docker and doesn't really fit in this repo. I think it's fine to just omit this. We rarely get vuln reports in this repo, and when we do they are mostly legit, so I don't think it's necessary to have this workflow defined.